docs/UserGuide.md

```diff
@@ -6,50 +6,93 @@
 
 # FApro User Guide
 
-FApro is a **desktop app for managing clients' contacts, optimized for use via a Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI).
+<!-- * Table of Contents -->
+<page-nav-print />
+
+## Welcome Note
+
+Greetings financial advisors, hope you are doing well! We would like to thank you for your services all these years. Your hard work has not gone unnoticed! Particularly, we have observed that you may be be facing issues in the following aspects:
+* **Client Engagement Tracking**: Difficulty in monitoring the service history of a large, diverse client base to ensure satisfaction.
+* **Appointment Management**: Significant time and effort spent on scheduling and remembering appointments.
+* **Document Accessibility**: Challenges in promptly recalling or locating specific customer portfolio documents.
+
+Thus, we have specially developed _FApro_ to aid you in your everyday tasks so that you are able to spend more time fostering **meaningful relationships** and providing **personalized service**.
+
+You may refer to the [Product Information](#product-information) section to find out more about _FApro_. If you are keen to use our product, we have a [Quick Start](#quick-start) dedicated to help you get started with ease. We hope you enjoy using _FApro_!
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Purpose of this User Guide
 
-If you
+Dear financial advisors, this User Guide will aid you in your journey when navigating _FApro_. We have included beginner-friendly instructions to help you get started, familiarise yourself with our various features. Additionally, we also have some examples to let you test out the product and tips to help you fully utilize our product. We hope you have an amazing time using our product!
```

Comment:
Should we still start off with "Dear ... " here? Sounds abit awkward since the UG already greeted them at the top. Maybe can start with "This User Guide will..."

Reply:
Sorry I might be a bit anal on this hahaha but I think too much of "the product" "our product" towards the end.
Maybe can shortern it.

"Additionally, we've included examples and tips to enhance your experience with our product. We hope you enjoy using it to its fullest potential!"